nebula_examples/CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ target_link_libraries(hesai_ros_offline_extract_pcd PUBLIC
     nebula_decoders::nebula_decoders_hesai
 )
 
```

Comment:
Is this needed? We were avoiding `ament_auto` on purpose for reduce building time